### ISSUE TITLE
[Bug] blocked-words from moderation.yml does not work

### ISSUE THREAD

**Issue by SrBedrock** [Hexaoxide/Carbon#40]
### Description

Messages sent in the chat containing the word specified in the configuration `blocked-words` are not blocked.

### How to reproduce

1. Add a word in the configuration;

```yml
filters:
  enabled: true
  case-sensitive: false
  filters:
  [...]
  # Anything in blocked-words will prevent the message from being sent at all.
  blocked-words:
      - "teste"
```

2. Restart the server;

3. Send the word in chat.

![image](https://user-images.githubusercontent.com/51332006/91279891-7427e300-e75c-11ea-9342-3f6fca3b13c2.png)

**Comment by Laarryy** [comment]
it's regex, so try (.*:?)?teste(.*:?)?

**Comment by SrBedrock** [comment]
> it's regex, so try (._:?)?teste(._:?)?

The message remains unblocked.

![image](https://user-images.githubusercontent.com/51332006/91348730-96e8e480-e7ba-11ea-8c17-cba6f6fc363a.png)

**Comment by Laarryy** [comment]
Oops, GitHub markdown got me! There are two asterisks - `(.*:?)?teste(.*:?)?`

**Comment by SrBedrock** [comment]
> Oops, GitHub markdown got me! There are two asterisks - `(.*:?)?teste(.*:?)?`

Did not work.

**Comment by Draycia** [comment]
You shouldn't need anything fancy, just `- "blocked thing here"`, I'll look into it later

**Comment by Draycia** [comment]
Fixed as of eaf172b
